scripts/release/release.js

```diff
@@ -0,0 +1,10 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+const shell = require('shelljs');
```

Comment:
Probably not important this go-around, but this release script is the same as terra-toolkit. The thing I notice there is manually releasing the repo often does not reflect in the changelog because there is not check.